pycon/settings.py

```diff
@@ -335,6 +335,7 @@
 
     'markitup',
     'cms_utils',
+    'social_django',
```

Comment:
what's social_django?

Reply:
I think it's part of `python-social-auth` that's been refactor the the separate package - the whole `python-social-auth` module is now depracated and has been split into a number of smaller, self contained packages. The `tests.test_cfp.TestCFP testMethod=test_665_edit_with_other_in_django_admin` test would fail without this change to the `settings.